website/docs/r/monitor_smart_detector_alert_rule.html.markdown

```diff
@@ -0,0 +1,109 @@
+---
+subcategory: "Monitor"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_monitor_smart_detector_alert_rule"
+description: |-
+  Manages an Monitor Smart Detector Alert Rule.
+---
+
+# azurerm_monitor_smart_detector_alert_rule
+
+Manages an Monitor Smart Detector Alert Rule.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_application_insights" "example" {
+  name                = "example-appinsights"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  application_type    = "web"
+}
+
+resource "azurerm_monitor_action_group" "example" {
+  name                = "example-action-group"
+  resource_group_name = azurerm_resource_group.example.name
+  short_name          = "exampleactiongroup"
+}
+
+resource "azurerm_monitor_smart_detector_alert_rule" "example" {
+  name                = "example-smart-detector-alert-rule"
+  resource_group_name = azurerm_resource_group.example.name
+  severity            = "Sev0"
+  scope               = [azurerm_application_insights.example.id]
+  frequency           = "PT1M"
+  detector_id         = "FailureAnomaliesDetector"
+
+  action_group {
+    ids = [azurerm_monitor_action_group.test.id]
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Monitor Smart Detector Alert Rule. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) Specifies the name of the resource group in which the Monitor Smart Detector Alert Rule should exist. Changing this forces a new resource to be created.
+
+* `detector_id` - (Required) Specifies the Built-In Smart Detector id that this alert rule will use. Currently possible values are `FailureAnomaliesDetector`.
+
+* `scope` - (Required) Specifies the scopes of this Smart Detector Alert Rule.
```

Comment:
could we be a bit more clear what type of value/values are epected here?

Reply:
the scope depends on smart detector. Different detector supports different scope